gogen/gogen.go

```diff
@@ -167,8 +167,10 @@ func (g GoStructCodeSnippet) String() string {
 
 // goStructField contains a definition of a field within a Go struct.
 type goStructField struct {
-	Name string // Name is the field's name.
-	Type string // Type is the Go type of the field.
+	// YANGName is the YANG name of the key field.
```

Comment:
Consistently keep this comment to the right, or change the others to be the line above?

Reply:
Done.